app/controllers/pages_controller.rb

```diff
@@ -1,2 +1,17 @@
 class PagesController < ApplicationController
+  include PageableFeedConcern
+
+  before_action :authenticate_user!
+
+  def home
+    respond_to do |format|
+      format.html { initialize_pageable_feed }
+      format.js {
+        @feed_item = FeedItemPresenter.new(
+          feed_item: current_user.toggle_feed_item_recommended(params[:id]),
+          current_user: current_user,
+        )
```

Comment:
NOTE: Maintainable. Use of presenters (really decorators in this case) to eliminate a million, messy, confusing @vars that are difficult to keep track of.